substrate/frame/support/procedural/src/pallet/expand/tt_default_parts.rs

```diff
@@ -198,9 +198,9 @@ pub fn expand_tt_default_parts(def: &mut Def) -> proc_macro2::TokenStream {
 		macro_rules! #default_parts_unique_id_v2 {
 			{
 				$caller:tt
-				frame_support = [{ $($frame_support:ident)::* }]
+				your_tt_return = [{ $my_tt_return:path }]
```

Comment:
@gupnik this is from your commit which I reverted, should they remain?

Reply:
Yes please. This was the fix for using the frame crate with the v2 macro.